src/test/erc4626/ERC4626.t.sol

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.10;
+
+import {Vm} from '../../../lib/forge-std/src/Vm.sol';
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import {DefiBridgeProxy} from './../../aztec/DefiBridgeProxy.sol';
+import {RollupProcessor} from './../../aztec/RollupProcessor.sol';
+import {IERC4626} from './../../bridges/erc4626/Interfaces/IERC4626.sol';
+import {VaultBridge} from './../../bridges/erc4626/VaultBridge.sol';
+import {AztecTypes} from './../../aztec/AztecTypes.sol';
+import {console} from '../console.sol';
+import {Test} from '../../../lib/forge-std/src/Test.sol';
+
+//Tested at block 14886873 may not work at other blocks
+contract ERC4626 is Test {
+    Vm vm = Vm(0x7109709ECfa91a80626fF3989D68f67F5b1DD12D);
+
+    DefiBridgeProxy defiBridgeProxy;
+    RollupProcessor rollupProcessor;
+
+    VaultBridge vaultbridge;
+
+    IERC20 constant maple = IERC20(0x33349B282065b0284d756F0577FB39c158F935e6);
+    IERC20 constant fei= IERC20(0xc7283b66Eb1EB5FB86327f08e1B5816b0720212B)
+    IERC4626 constant tribeVault = IERC4626(0x4937A209D4cDbD3ecD48857277cfd4dA4D82914c);
+    IERC4626 constant vault = IERC4626(0x4937A209D4cDbD3ecD48857277cfd4dA4D82914c);
+
+    function _aztecPreSetup() internal {
+        defiBridgeProxy = new DefiBridgeProxy();
+        rollupProcessor = new RollupProcessor(address(defiBridgeProxy));
+    }
+
+    function setUp() public {
+        _aztecPreSetup();
+
+        vaultbridge = new VaultBridge(address(rollupProcessor));
+        //vaultbridge.validateVault(address(vault));
+        rollupProcessor.setBridgeGasLimit(address(vaultbridge), 1000000);
+    }
+
+    function testVaultBridge1() public {
```

Comment:
Please use property based testing (fuzzing) as part of the tests. Example here would be to have `depositAmount` as input, and then make it bound to be within some range.
```solidity
function testVaultBridge1(uint96 _depositAmount) public {
  uint256 depositAmount = bound(_depositAmount, 1000, type(uint96).max);
```